backend/src/akvo/lumen/component/keycloak.clj

```diff
@@ -93,6 +110,32 @@
         (response {:status (:status ed)
                    :body (:reasonPhrase ed)})))))
 
+(defn tenant-admin?
+  [request-draft api-root tenant user-id]
```

Comment:
Request-draft is a request object that is used with clj-http (the second argument to request functions). Thinking about it in hindsight I do think this is a questionable design. I should have named it request-headers and passed that around. Would have been just as simple to build a map with {:headers <request-headers> :body <body>} as what I do now (assoc request-draft :body <body>}.